config/samples_Run2_2018.yaml

```diff
@@ -33,12 +33,52 @@ GLOBAL:
       VSe: VVLoose
       VSmu: VLoose
       VSjet: Medium
+  corrections:
+    - tauID
+    - tauES
+  deepTauVersion: 2p5
```

Comment:
for the moment keep 2p1 as default value

Reply:
ok